src/Core/Checkout/Cart/RedisCartPersister.php

```diff
@@ -51,7 +51,7 @@ public function load(string $token, SalesChannelContext $context): Cart
         }
 
         try {
-            $value = \unserialize($value);
+            $value = @\unserialize($value);
```

Comment:
why do we need to silence that call? this could suppress underlying issues :thinking: 

we should at least add a comment

Reply:
$value is checked afterwards. it's fine then

Reply:
and/or explicitly check for null or whatever unserialize returns in case of error

What error was thrown here that we want to prevent? Can we catch it? As we "only" catch exception, but not throwable here?

Reply:
ok isSet is pretty solid
https://onlinephp.io/c/ba4f5

as it will work as expected for null, false, empty string it should be solid